pkgs/development/tools/cbfmt/default.nix

```diff
@@ -22,5 +22,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/lukas-reineke/cbfmt";
     license = licenses.mit;
     maintainers = [ maintainers.stehessel ];
+    platforms = platforms.all;
```

Comment:
This is actually not useful - `rustPlatform.buildRustPackage` defaults to `rustc.meta.platforms`, which is `platforms.linux ++ platforms.darwin`

```suggestion
```